projectEuler/problem6.js

```diff
@@ -0,0 +1,18 @@
+// prompt: find the difference between the sum of the squares of the first one hundred natural numbers and the square of the sum
+
+function sumSquareDiff() {
+  let i = 1;
+  var a = 0;
+  var b = 0;
```

Comment:
Prefer `let` instead of var in these two variables. You may also want to use self-documenting names for these variables instead of a generic letter. `a` could be `sumOfValues`, `b` could be `sumOfSquaredValues`.